docs/how-to/distributed-training.txt

```diff
@@ -47,6 +47,43 @@ Example configuration with distributed training:
    resources:
      slots_per_trial: N
 
+
+Data Downloading
+-------------------
+
+When performing distributed training, Determined will automatically
+create one process for every GPU that is being used for training. Each
+process will attempt to download training and/or validation data, so
+care should be taken to ensure that concurrent data downloads do not
+conflict with one another. One way to do this is to include a unique
+identifier in the local file system path where the downloaded data is
```

Comment:
It could be a shared file system path. That's why we use `rank` not `local_rank`.

Reply:
Well, if the file system path is shared, even `rank` is not sufficient to avoid conflicts between multiple experiments.

Reply:
That's a fair point. I think we should also explain it can't conflict with other experiments here and to what extent should not the path have conflict.

Reply:
Well, any time you're writing to shared storage (whether using dtrain or not), you need to be aware of possible conflicts with other writers -- i.e., not dtrain specific. So while it doesn't hurt to call this out, I feel like the main focus here should be on the stuff Determined does here that is a bit surprising / happens automatically (spawning multiple processes and downloading data concurrently).